e2e/libs/k8s/k8s.py

```diff
@@ -80,6 +80,31 @@ def wait_all_pods_evicted(node_name):
 
     assert evicted, 'failed to evict pods'
 
+def get_node_status(node_name):
+    api = client.CoreV1Api()
+    node = api.read_node(node_name)
+    conditions = node.status.conditions
+    for condition in conditions:
+        if condition.type == "Ready" and condition.status == "True":
+            return True
+    return False
+
+def get_node_status_debug(node_name):
```

Comment:
Is this function required? If so, could we try to merge the debug messages into the `get_node_status` function?

Reply:
> Is this function required? If so, could we try to merge the debug messages into the `get_node_status` function?

This function was used during the development process, but I forgot to remove it. I will remove it.